lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -963,10 +971,17 @@ public function usesSequenceEmulatedIdentityColumns()
      */
     public function getIdentitySequenceName($tableName, $columnName)
     {
-        $table = new Identifier($tableName);
+        $table                = new Identifier($tableName);
+        $identitySequenceName = $table->getName();
+
+        // max possible length without '_SEQ' suffix
+        $maxPossibleLengthWithoutSuffix = $this->getMaxIdentifierLength() - 4;
```

Comment:
Could this repeating code be extracted in a separate private method e.g. `addSuffix(string $identifier, string $suffix): string`?